src/query/service/src/sql/planner/plans/aggregate.rs

```diff
@@ -94,12 +94,8 @@ impl PhysicalOperator for Aggregate {
                     required.distribution = Distribution::Any;
                 } else {
                     // Group aggregation, enforce `Hash` distribution
-                    required.distribution = Distribution::Hash(
-                        self.group_items
-                            .iter()
-                            .map(|item| item.scalar.clone())
-                            .collect(),
-                    );
+                    required.distribution =
+                        Distribution::Hash(vec![self.group_items[0].scalar.clone()]);
```

Comment:
@zhang2014 , why use the first groupby item not all items here?

Reply:
implementation of combined hash is very slow. 

By the way, we don't use this for shuffling, we always use virtual column(_group_by_key) to shuffle data.